### ISSUE TITLE
macOS brew install llvm@8 failed

### ISSUE THREAD

**Issue by ImAlien** [cmu-db/bustub#173]
error information:
```shell
llvm@8: The x86_64 architecture is required for this software.
Error: llvm@8: An unsatisfied requirement failed this build.
```

**Comment by coder-freestyle** [comment]
You can try changing the llvm version to 11 in packages.sh file. @ImAlien

**Comment by zhibin-huang** [comment]
You can use brew to install llvm@8 in x86 mode with Rosetta2.

**Comment by garrisonhess** [comment]
Resolved by #243
